modules/cluster-logging-elasticsearch-audit.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-elasticsearch.adoc
+
+[id="cluster-logging-elasticsearch-audit_{context}"]
+= Forward audit logs to the log store
+
+Because the internal {product-title} Elasticsearch log store does not provide secure storage for audit logs, by default audit logs are not stored in the internal Elasticsearch instance. 
+
+If you want to send the audit logs to the internal log store, for example to view the audit logs in Kibana, you must use the Log Forward API.
+
+[IMPORTANT]
+====
+The internal {product-title} Elasticsearch log store does not provide secure storage for audit logs. We recommend you ensure that the system to which you forward audit logs is compliant with your organizational and governmental regulations and is properly secured. {product-title} cluster logging does not comply with those regulations.
+====
+
+.Procedure
+
+To use the Log Forward API to forward audit logs to the internal Elasticsearch instance:
+
+. If the Log Forward API is not enabled:
+
+.. Edit the Cluster Logging Custom Resource (CR) in the `openshift-logging` project:
++
+----
+$ oc edit ClusterLogging instance
+----
+
+.. Add the `clusterlogging.openshift.io/logforwardingtechpreview` annotation and set to `enabled`:
++
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  annotations:
+    clusterlogging.openshift.io/logforwardingtechpreview: enabled <1>
+  name: "instance"
+  namespace: "openshift-logging"
+spec:
+
+...
+
+  collection: <2>
+    logs:
+      type: "fluentd"
+      fluentd: {}
+----
+<1> Enables and disables the Log Forwarding API. Set to `enabled` to use log forwarding. To use the only the {product-title} Elasticsearch instance, set to disabled or do not add the annotation.
+<2> The `spec.collection` block must be defined to use Fluentd in the Cluster Logging CR.
+
+. Create a Log Forwarding CR YAML file or edit your existing CR similar to the following:
++
+* Create a CR to send all log types to the internal Elasticsearch instance. You can use the following example without making any changes:
++
```

Comment:
@mburke5678  Shall we mention the 'TechPreview LogForward' in 4.6?

Reply:
From Technique, LGTM, If we won't merge TP LF when backporting to 4.6.

Reply:
@anpingli Thank you for staying up late and reviewing. Yes, this PR is for 4.5 Tech Preview and I will update for the GA Log Forward API.